src/sharers/viber.js

```diff
@@ -10,5 +11,5 @@ export default function viber(options = {}) {
     text: [title, url].filter(item => item).join(' '),
   });
 
-  return window.location.assign(`viber://forward?${params}`);
+  return mobileShare(`viber://forward?${params}`);
```

Comment:
You need to check mobileShare call instead of location.assign in the sharers/viber test